app/Models/Journal/Entry.php

```diff
@@ -75,6 +75,7 @@ public function getInfoForJournalEntry()
             'month_name' => mb_convert_case(DateHelper::getShortMonth($this->date), MB_CASE_UPPER, 'UTF-8'),
             'year' => $this->date->year,
             'date' => $this->date,
+            'created_at' => $this->created_at->format('G:i j M Y'),
```

Comment:
You should format the date with DateHelper::getShortDateWithTime
```suggestion
            'created_at' => DateHelper::getShortDateWithTime($this->created_at),
```